packages/component-library/src/components/plmg-page-container/plmg-page-container.scss

```
@@ -9,7 +9,7 @@
   .plmg-content {
     display: block;
     position: absolute;
-    height: 100%;
+    height: calc(100% - 99px); //Height of the header
```

```suggestion
    height: calc(100% - (51px + 2 * tokens.$plmg-spacing-x1-5);;```
Looks right the right fix. Only suggestion would be to maybe use the spacing token here in case that changes at some point?

---

Looks right the right fix. Only suggestion would be to maybe use the spacing token here in case that changes at some point?

---

And I guess the same logic would then apply to line 13? I looked at where the 99px is coming from and I believe its calculated in the .plmg-header-container `height: 99px - 2 * tokens.$plmg-spacing-x1-5;`

---

Sidenote: My comment in the code here was weird, i should have said `subtract height of the header`.

Yeah I think this is a bit confusing because it seems plumage is using[ `box-sizing: content-box` ](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing)instead of `border-box`. `content-box` is the default but normally I see that being set to `border-box` immediately in most projects I've worked in :p 

What that means is, when we set `width` or `height` for an element in with `box-sizing: content-box`, it's setting it for the **content** only. It doesn't include the padding (or border) width/height in the width/height of the element you're setting.

In our case, the height of the header **including** the padding is 99px, so I'm subtracting that entire height from the page content height. But, when we actually set the height of the header in the css for the header component, we're setting the height of just the header content. That's why we start off with 99px (the total height including padding) and subtract the padding from it.

I feel like this would be very confusing to read so if you want to chat about it let me know 😆 

All that being said, having this be a variable as you mentioned would be better :)

---

** some components are using border-box, some aren't 😬

---

👍 Thanks. I think I get it now but let's have a chat about it on Monday!